browser/ui/webui/speedreader/speedreader_toolbar_data_handler_impl.cc

```diff
@@ -139,10 +144,12 @@ void SpeedreaderToolbarDataHandlerImpl::AiChat() {
   if (!browser_ || !browser_->window()) {
     return;
   }
+#if BUILDFLAG(ENABLE_AI_CHAT)
   if (auto* ai_chat_tab_helper = ai_chat::AIChatTabHelper::FromWebContents(
```

Comment:
Should this also check for feature flag? `ai_chat::features::IsAIChatEnabled()`

Reply:
Yes probably since AIChatTabhelper doesn't check it. Also what's the point of assigning `ai_chat_tab_helper`?

Reply:
We definitely can - my gut is that it's outside the scope of this issue (fixing the Release build). But if you were able to add a follow up commit, I'd be 👍